index.html

```diff
@@ -0,0 +1,229 @@
+            <div class="Card3-tech">
+              <div>
+                <p class="P1">Canopy</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P2">Back End Dev</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P3">2015</p>
+              </div>
+            </div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button">See projects</button>
+        </div>
+        <div class="Card4">
+          <img src="img/Snapshoot Portfolio (3).png" alt="" />
+          <div class="Card4-P">
+            <h4>Multi-Post Stories</h4>
+            <div class="Card4-tech">
+              <div>
+                <p class="P1">Canopy</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P2">Back End Dev</p>
+                <img src="img/Counter.png" alt="" />
+                <p class="P3">2015</p>
+              </div>
+            </div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button">See projects</button>
+        </div>
+      </section>
+      <section class="About">
+        <div class="Myself">
+          <h2>
+            About <br />
+            Myself
+          </h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h3>LET'S CONNECT</h3>
+          <ul>
+            <a href="https://twitter.com/SolomonAfutu"
+              ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+              ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href="https://github.com/SolomonTheCoder"
+              ><i class="fa fa-github fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href="">
+              <i class="fa fa-medium fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href=""
+              ><i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i
+            ></a>
+          </ul>
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="Language">
+          <h2>
+            Languages <i class="fa fa-angle-down fa-2x" aria-hidden="true"></i>
+          </h2>
```

Comment:
- [ ] Kindly fix the drop-down arrow

- [ ] Your design
<img width="501" alt="Screenshot 2023-03-23 at 16 18 28" src="https://user-images.githubusercontent.com/81444338/227385859-21c6c463-48e3-495d-acfa-eec4532ffae6.png">

- [ ] Figma Design
<img width="624" alt="Screenshot 2023-03-23 at 16 19 59" src="https://user-images.githubusercontent.com/81444338/227385906-df901bf3-f098-48b6-982a-547ad8167ed2.png">